x-pack/plugins/security/public/management/api_keys/api_keys_grid/empty_prompt/empty_prompt.tsx

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
This file should be removed - it's trying to place it into the NP version of the client-side security plugin, which didn't really exist in `7.6`